pkg/controllers/cloud/configmap_controller.go

```diff
@@ -86,11 +93,36 @@ func (r *ConfigMapReconciler) setCloudResourcePrefix(prefix string) error {
 	return nil
 }
 
+// setCloudSyncInterval sets the CloudSyncInterval, returns error when invalid.
+func (r *ConfigMapReconciler) setCloudSyncInterval(interval int64) error {
+	if interval == 0 {
+		r.Log.Info("Setting CloudSyncInterval to default", "Old", r.ControllerConfig.CloudSyncInterval,
+			"New", config.DefaultCloudSyncInterval)
```

Comment:
Drop old with default.. same goes for prefix

Reply:
Solved.